README.md

```diff
@@ -128,35 +61,182 @@ You are now ready to clone the repository and run `npm install`.
    npm config set msbuild_path <path/to/MSBuild.exe>
    ```
 
+You can now clone the repository and run `npm install`.
+
 [Scoop]: https://scoop.sh/
 [Visual Studio docs]: https://docs.microsoft.com/en-us/visualstudio/install/modify-visual-studio?view=vs-2022
 [Windows Subsystem for Linux (WSL)]: https://docs.microsoft.com/en-us/windows/wsl/install
 
-### How To Run
 
-Use the following commands. The former is needed the first time or after an
-update is pulled from upstream. The latter is needed for follow-up starts.
+### macOS and Linux
+
+Install [Node.js][Node.js] v16. Then you can install dependencies with:
 
 ```
-npm install
-npm run dev
+npm run install
 ```
 
-To build the distributable (application bundle on macOS, installer on Windows),
-run `npm run build`.
+[Node.js]: https://nodejs.org/
+
+
+## Running
+
+Once you have your dependencies installed you can run a development version
+of Rancher Desktop with:
 
-On Linux `npm run build` produces a zip file including the built binaries. To build the 
-distributable artifacts (RPM, Deb or AppImage) the [Open Build Service] is used.
-OBS makes use of the packaging recipes under `packaging/linux` folder of this
-repository together with the zip file including all built binaries.
+```
+npm run dev
+```
 
-[Open Build Service]: https://build.opensuse.org/
 
-### How To Test
+## Tests
 
-Use the following commands to run unit tests and e2e tests.
+To run the unit tests:
 
 ```
 npm test
+```
+
+To run the integration tests:
+
+```
 npm run test:e2e
 ```
+
+
+## Building
+
+Rancher can be built from source on Windows, macOS or Linux.
+Cross-compilation is currently not supported.
+
+
+### Windows and macOS
+
+```
+npm run build
+```
+
+The build output goes to `dist/`.
+
+
+### Linux
+
+On Linux it is not possible to completely build Rancher Desktop from
```

Comment:
We still get a zip file on Linux from `npm run build`, right? It's only packaged builds (rpm, AppImage, etc.) that require OBS?

Reply:
Hmm you make a good point. I suppose that someone could unpack the zip file and use the application inside. Though I just tried it and it hung at the "Waiting for services" step. We could tell people that `npm run build` will produce a zip file that can be unzipped to get a runnable application, but if there are issues I'm not sure it's worth it. What do you think?

Reply:
The zip build seems to work for me.

Of course, I'm trying this in WSL (using the Linux binaries in a Linux distribution), so… things can be a bit weird :D

Reply:
Ok well I'll add a sentence about the zip build. If we end up having complaints because it doesn't work properly we can treat it as a low-priority bug.